src/tasks/level10.ts

```diff
@@ -156,7 +156,7 @@ export const GiantQuest: Quest = {
       combat: new CombatStrategy().startingMacro(
         Macro.trySkill($skill`%fn, let's pledge allegiance to a Zone`)
       ),
-      choices: { 670: 5, 669: 1, 671: 4 },
+      choices: { 670: 1, 669: 1, 671: 4 },
```

Comment:
There is no point to grabbing the dumbell if we have an amulet, and it slows us down.

Reply:
I changed this logic... a lot. We now do whatever we can to get through the NC fast.